Tella/Scenes/HomeView/ViewModel/FileListViewModel.swift

```diff
@@ -367,6 +367,23 @@ extension FileListViewModel {
         }
     }
     
+    
+    func checkReportsBeforeDelete() -> Bool {
+        // check reports before deleting files
+        
+        let fileIds = selectedFiles.compactMap { $0.id }
+        let fileIsInReport = appModel.tellaData?.database.checkFilesInReports(ids: fileIds)
+
+        if fileIsInReport == true {
+            // Show pop up
+            return  true
+        } else {
+            deleteSelectedFiles()
```

Comment:
I think `deleteSelectedFiles()` will directly delete the file after user clicks on delete file